commands/ipblock.go

```diff
@@ -186,11 +187,18 @@ func RunIpBlockGet(c *core.CommandConfig) error {
 }
 
 func RunIpBlockCreate(c *core.CommandConfig) error {
+	name := viper.GetString(core.GetFlagName(c.NS, config.ArgName))
+	location := viper.GetString(core.GetFlagName(c.NS, config.ArgLocation))
+	size := viper.GetInt32(core.GetFlagName(c.NS, config.ArgSize))
+	c.Printer.Infof("Properties set for creating the Ip block: Name: %v, Description: %v, Size: %v", name, location, size)
```

Comment:
Mismatch between parameters: description and location

Reply:
done